src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1267,6 +1267,9 @@ internal void AddAndRegisterNode(NodeModel node, bool centered = false)
 
             HasUnsavedChanges = true;
 
+            if (node is CodeBlockNodeModel cbn
+                && string.IsNullOrEmpty(cbn.Code)) return;
```

Comment:
Unrelated - this just prevents an unnecessary graph run when double-clicking on canvas to create an empty CBN.

Reply:
don;t know if this is more readable but it is a bit shorter
```
if (string.IsNullOrEmpty((node as CodeBlockNodeModel)?.Code))
```